README.md

```diff
@@ -101,4 +101,4 @@ For topics that are better discussed live, please join the
 
 ## License
 
-This project is licensed under the Apache License 2.0 - see the [LICENSE.md](LICENSE.md) file for details
+This project is licensed under the Apache License 2.0 - see the [LICENSE.md](LICENSE) file for details
```

Comment:
instead i suggest renaming the file to `LICENSE.md` and keep readme as it is now

Reply:
done